interface-definitions/firewall.xml.in

```diff
@@ -383,6 +383,26 @@
                 </children>
               </node>
               #include <include/firewall/common-rule.xml.i>
+              <leafNode name="ip-length">
```

Comment:
As to me it should be “packet-length”

Reply:
Since in ipv4 its "total length" and in ipv6 is "payload length", I chose "ip-length" so it's user-friendly, and same option in both protocols.

I think if changes are necessary, I would go for those values, instead of "packet-length". But of course, open to discussion and more opinions.
@dmbaturin @c-po @zdc

Reply:
+1 on `packet-length`, as `ip` is sometimes associated with IPv4.

Reply:
Same here. `ip-length` also doesn't really help to resolve the confusion between IPv4 and IPv6 behavior, and most people will associate "IP" with "IPv4", so it will only create more confusion.